task26.py

```diff
@@ -0,0 +1,9 @@
+import random
+a = int(random.randrange(0,10))
+b = int(random.randrange(1,10))
+print(a,b)
+def pip(a,b):
+    if a in (0,1) or b==1:
+        return a
+    return a*pip(a,b-1)
+print(pip(a,b))
```

Comment:
Не забывайте про PEP8
```suggestion
import random

a = int(random.randrange(0, 10))
b = int(random.randrange(1, 10))
print(a, b)


def pip(a, b):
    if a in (0, 1) or b == 1:
        return a
    return a * pip(a, b - 1)


print(pip(a, b))
```

Reply:
Что-то не так
![image](https://user-images.githubusercontent.com/49741194/223320625-da489464-7080-4296-beee-425571a0527c.png)
А еще, если вторым аргументом передать 0, то произойдет переполнение стека. Но нулевой показатель степени вполне может быть.